pkg/scheduler/core/generic_scheduler.go

```diff
@@ -166,7 +166,7 @@ func (g *genericScheduler) selectClusters(clustersScore framework.ClusterScoreLi
 	return spreadconstraint.SelectBestClusters(placement, groupClustersInfo, spec.Replicas)
 }
 
-func (g *genericScheduler) assignReplicas(
+func (g *genericScheduler) AssignReplicas(
```

Comment:
Is there anywhere that need to call this func?

Reply:
Is it sufficient to only export this method based on the description of the issue?